plugins/gameobjects/tagtext/tagtext/TagText.d.ts

```diff
@@ -7,7 +7,7 @@ declare namespace TagText {
     interface TagProg {
         color?: string,
 
-        stroke: {
+        stroke?: {
```

Comment:
Hey there!

I've noticed that stroke was not marked as optional, but it is. If I'm correct, here's the default values for the stroke:

https://github.com/rexrainbow/phaser3-rex-notes/blob/master/plugins/gameobjects/textbase/textstyle/PropertyMap.js#L20

Also, I've tried not to pass `stroke` at all, and everything worked perfectly. But for now, due to the fact that the field is not marked as optional, I had to pass it like `stroke: {}` in my project 🐨